### PR TITLE
added column_names_to_upper/lower methods to Metadata class

### PR DESCRIPTION
**Pull request by LavMatt**

### CODE REVIEW

File: mojap_metadata/metadata/metadata.py [hunk 1]
```diff
@@ -424,6 +424,28 @@ def to_yaml(self, filepath: str, mode: str = "w", **kwargs) -> None:
         with open(filepath, mode) as f:
             yaml.safe_dump(self.to_dict(), f)
 
+    def column_names_to_lower(self, inplace=False) -> object:
```
Comment (s-ducks) [inline mojap_metadata/metadata/metadata.py:427]:
```suggestion
    def column_names_to_lower(self, inplace: bool=False) -> Union[object, None]:
```

File: mojap_metadata/metadata/metadata.py [hunk 2]
```diff
@@ -424,6 +424,28 @@ def to_yaml(self, filepath: str, mode: str = "w", **kwargs) -> None:
...
+            for c in self.columns:
+                c['name'] = c['name'].lower()
+            return self
+        else:
+            self_lower = deepcopy(self)
+            for c in self_lower.columns:
+                c['name'] = c['name'].lower()
+            return self_lower
+
+    def column_names_to_upper(self, inplace=False) -> object:
```
Comment (s-ducks) [inline mojap_metadata/metadata/metadata.py:438]:
```suggestion
    def column_names_to_upper(self, inplace: bool=False) -> Union[object, None]:
```

**Review by s-ducks — COMMENTED** [review]
For the tests it's better to have minimal (preferably 1 ) assertion per test
This would mean having more tests that are more specific, and hopefully more parameterisable (so covering more cases)

File: tests/test_metadata.py [hunk 3]
```diff
@@ -17,6 +17,50 @@
...
+            {"name": "A", "type": "int8"},
+            {"name": "B", "type": "string"},
+            {"name": "C", "type": "date32"},
+            {"name": "D", "type": "date32"},
+            {"name": "E", "type": "date32"},
+        ]
+    )
+    return meta
+
+
```
[the remark this answers is not in the document]
Comment (LavMatt) [reply tests/test_metadata.py]: done

**Review by s-ducks — APPROVED** [review]

**Pull request merged**